packages/blockchain-wallet-v4/src/redux/common/bch/selectors.js

```diff
@@ -18,8 +18,9 @@ export const getActiveHDAccounts = state => {
   const addInfo = account => balancesRD.map(prop(prop('xpub', account)))
     .map(x => assoc('info', x, account))
   const addBchLabel = account => account.map(a => assoc('label', path([prop('index', a), 'label'], bchAccounts), a))
+  const addArchived = account => account.map(a => assoc('archived', path([prop('index', a), 'archived'], bchAccounts), a))
 
-  const objectOfRemotes = compose(map(addBchLabel), map(addInfo),
+  const objectOfRemotes = compose(map(addArchived), map(addBchLabel), map(addInfo),
```

Comment:
@mpfluger this will determine based on bchAccounts (metadata) whether or not bch account is archived. Later frontend will use this to filter active accounts https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/438/files#diff-85554d9a8902a63ce733de1601493254R7

Reply:
That sounds right but @jtormey has more experience with that. Could you give it a quick look as well please?

Reply:
Trying to understand what's happening. Are we computing the BCH accounts by copying the BTC accounts and adding relevant BCH information (label, archived)? If so then yeah this looks fine. Although, my preference would probably be to start with BCH accounts and add BTC data as needed.